src/main/kotlin/no/nb/mlt/wls/application/synqapi/synq/SynqController.kt

```diff
@@ -52,13 +58,43 @@ class SynqController(
         )
     )
     @PutMapping("/item-update")
-    suspend fun updateItem(
+    suspend fun moveItem(
```

Comment:
Maybe rename the endpoint to /move-item as well? That way it will be more consistent with function name and define the scope better.